### PR TITLE
Update ch01.asciidoc

### PR DESCRIPTION
**Pull request by bitcoina**
Just added a couple of points for consideration in the history.   I propose that Christian Decker should be recognized for his work on a separate lightning paper, which was eventually not released, as well as Tadge and Joseph, who made the definitive software.   

Also initial node software, the "reckless" movement and lightning.place are all important parts of the history.

### CODE REVIEW

**Comment by bitcoina** [comment]
Just added a couple of points for consideration in the history. I propose that Christian Decker should be recognized for his work on a separate lightning paper, which was eventually not released, as well as Tadge and Joseph, who proposed the "definitive" white paper. 

Also initial node software, the "reckless" movement and lightning.place are all important parts of the history.

File: ch01.asciidoc [hunk 1]
```diff
@@ -19,12 +19,14 @@ Due to the use of onion routing, which is also used in tor, even the nodes that
 
 * Satoshis nSequence number
 * unidirectional payment channels
-* white paper
+* Multiple white papers (Joseph Poon, Thaddeus Dryja - https://lightning.network/lightning-network-paper.pdf) / Christian Decker
```
Comment (bitcoina) [inline ch01.asciidoc]: https://tik-old.ee.ethz.ch/file/716b955c130e6c703fac336ea17b1670/duplex-micropayment-channels.pdf

**Comment by bitcoina** [comment]
re-opening - will add the link to Christian Deckers proposal later.

File: ch01.asciidoc [hunk 2]
```diff
@@ -19,12 +19,14 @@ Due to the use of onion routing, which is also used in tor, even the nodes that
...
 * Milan meeting and creation of BOLTs
 * segwit activation
 * passing of integration tests / mainnet launch
 * Australia Meeting and BOLT 1.1
-* The lightning torch
+* Initial nodes/wallets - eclair, c-lightning etc
+* Reckless - Testing on mainchain.
+* Lightning.place / The lightning torch
```
Comment (renepickhardt) [inline ch01.asciidoc]: You meant satoshis.place? What was lightning.place?did I miss something?

**Review by renepickhardt — COMMENTED** [review]
Don't know lightning.place. Please clarify.

Rest seems certainly worthwhile mentioning

File: ch01.asciidoc (hunk 2 above)
Comment (bitcoina) [reply ch01.asciidoc]: Yeah, I meant satoshis.place.  :)  I think that was the moment a lot of people realized that lightning actually worked and the full power of it.

Comment (bitcoina) [reply ch01.asciidoc]: updated!  and I learnt more about how to use Github also...   Thanks :)

**Pull request merged**